independent-publisher-connectors/SOS Inventory/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2218 @@
+												"type": "boolean",
+												"description": "expired"
+											},
+											"customFields": {
+												"type": "array",
+												"description": "Custom Fields",
+												"items": {
+													"type": "object",
+													"properties": {
+														"id": {
+															"type": "integer",
+															"description": "ID"
+														},
+														"name": {
+															"type": "string",
+															"description": "Name"
+														},
+														"value": {
+															"type": "string",
+															"description": "Value"
+														},
+														"dataType": {
+															"type": "string",
+															"description": "Data Type"
+														}
+													}
+												}
+											},
+											"onhand": {
+												"type": "number",
+												"description": "On Hand"
+											},
+											"available": {
+												"type": "number",
+												"description": "Available"
+											},
+											"summaryOnly": {
+												"type": "boolean",
+												"description": "Summary Only"
+											}
+										}
+									}
+								}
+							}
+						}
+					}
+				},
+				"summary": "Lots",
+				"description": "List Lots",
+				"operationId": "Lot",
+				"x-ms-visibility": "important",
+				"parameters": [{
+						"in": "query",
+						"name": "location",
+						"type": "string",
+						"x-ms-summary": "Location",
+						"description": "Inventory at given location",
+						"required": false
+					},
+					{
+						"in": "query",
+						"name": "archived",
+						"type": "string",
+						"x-ms-summary": "Archived",
+						"description": "Archived",
+						"enum": [
+							"Yes",
+							"No"
+						],
+						"required": false
+					},
+					{
+						"in": "query",
+						"name": "status",
+						"description": "Expired, Recalled, Neither",
+						"type": "string",
+						"x-ms-summary": "Status",
+						"enum": [
+							"expired",
+							"recalled",
+							"notexpirednotrecalled"
+						],
+						"required": false
+					},
+					{
+						"in": "query",
+						"name": "limit",
+						"x-ms-summary": "Limit",
+						"description": "Max records to return, default 10",
+						"type": "integer",
+						"default": 10
+					}
+				]
+			}
+		},
+		"/location": {
+			"get": {
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"count": {
+									"type": "integer",
+									"description": "# Locations Returned"
+								},
+								"totalCount": {
+									"type": "integer",
+									"description": "# Matching Locations"
+								},
+								"data": {
+									"type": "array",
+									"items": {
+										"type": "object",
+										"properties": {
+											"id": {
+												"description": "Location ID",
+												"type": "integer"
+											},
+											"syncToken": {
+												"description": "syncToken",
```

Comment:
@Harold-Anderson strings in CaMel case are considered non-translatable. If you want to keep it as it is, then it's fine. Otherwise, we can change the value of this description to "Sync token".